test/Microsoft.ML.TestFramework/Datasets.cs

```diff
@@ -157,7 +157,7 @@ public static class TestDatasets
             name = "wine",
             trainFilename = "external/winequality-white.csv",
             testFilename = "external/winequality-white.csv",
-            loaderSettings = "col=Label:R4:11 col=Features:R4:0-10 sep=; header+"
+            loaderSettings = "col=Label:R4:11 col=Features:R4:0-10 sep=semicolon header+"
```

Comment:
>semicolon [](start = 71, length = 9)

Interesting... do we actually not support using `;` directly anymore? Not important, just curious.

Reply:
it working fine, not sure how this leaked to this pr

---
In reply to: [197503410](https://github.com/dotnet/machinelearning/pull/379#discussion_r197503410) [](ancestors = 197503410)